feature-libs/checkout/components/components/delivery-mode/delivery-mode.component.spec.ts

```diff
@@ -199,6 +203,29 @@ describe('DeliveryModeComponent', () => {
     ).toHaveBeenCalledTimes(1);
   });
 
+  describe('Shipping method fieldset', () => {
+    const getShippingMethodFieldSet = () =>
+      fixture.debugElement.query(By.css('fieldset'));
+
+    it('should be enabled after supported delivery modes are loaded', () => {
+      component.ngOnInit();
+      component.deliveryModeSetInProcess$ = of(false);
```

Comment:
```suggestion
      deliveryModeSetInProcess$.next(false);
```
the same in other places below